src/AppBundle/Manager/UserManager.php

```diff
@@ -327,6 +327,30 @@ public function getCategoriesExactMatchByUser(User $user, User $currentUser)
             $categories     = implode(', ', $categories);
         }
         return $categories;
+    }
+	/**
+     * @param User $user
+     * @param User $currentUser
+	 * @param $locale
+     * @return string
+     */
+    public function getCategoriesExactMatchByUserAndLocale(User $user, User $currentUser, $locale)
```

Comment:
This method name is almost identical to `getCategoriesExactMatchByUser` but returns a different result. Can you rename one?

Reply:
Tried to rename the fucntion. Is it ok now?